docs/Developers/Shortlinks.rst

```diff
@@ -0,0 +1,69 @@
+Shortlinks
+==========
+
+This page contains some useful shortlinks.
+
+.. Start ignoring LineLengthBear
+
+.. container :: shortlink-page
+
+  ==================  ===============================================================================================================================================
+  Shortlink           Full link
+  ==================  ===============================================================================================================================================
+  newcomer            https://api.coala.io/en/latest/Developers/Newcomers_Guide.html
+  newcomers           https://api.coala.io/en/latest/Developers/Newcomers_Guide.html
+  new                 https://github.com/issues?q=is%3Aopen%20is%3Aissue%20user%3Acoala%20label%3Adifficulty/newcomer%20-label%3Ainitiatives/gci%20no%3Aassignee%20
```

Comment:
GitHub full links do not look pretty particularly. It would be great if redirects links would be set up(after this is merged) for them just as https://coala.io/new/ is set up for all newcomer issues.